tools/setup_perf_cluster.sh

```diff
@@ -187,33 +206,6 @@ function get_fortio_k8s_ip() {
   echo "+++ In k8s fortio external ip: http://$FORTIO_K8S_IP:8080/fortio/"
 }
 
-function setup_istio_addons_ingress() {
```

Comment:
is this working some ofher way now?

Reply:
Seems that `grafana` is still not working